openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    pprint(settings.dict())
</pre><p>If I remember correctly, you shouldn't need to do this anymore: <a href="https://github.com/OpenFreeEnergy/gufe/blob/main/gufe/settings/models.py#L47" target="_blank">https://github.com/OpenFreeEnergy/gufe/blob/main/gufe/settings/models.py#L47</a></p><p> </p><p>I.e. the dictionary contents should be printed out by default (hopefully)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>